doc/tutorials/platformer/app/lib/ember_quest.dart

```diff
@@ -35,7 +35,7 @@ class EmberQuestGame extends FlameGame
       'star.png',
       'water_enemy.png',
     ]);
-    initializeGame(true);
+    initializeGame(loadHud: true);
```

Comment:
@munsterlander could you have a look at the EmberQuest changes and see if anything needs to be updated in the markdown too?

Reply:
Will do. I'm on travel for work but when I get to the hotel I will see what I can do from my tablet.

Reply:
Ok, all looks good for the files that were changed. I will need to add a pr then to update the code within the tutorial itself. I'm back in Saturday and can get it done over the weekend.

Reply:
Great, thanks!